EventNames.json should be part of the WebCore Xcode project

aprotyas opened the pull request:
#### 39dfb46486ffc25586e4a5b7f7f5a0e077da2ee8
<pre>
EventNames.json should be part of the WebCore Xcode project
<a href="https://bugs.webkit.org/show_bug.cgi?id=267791">https://bugs.webkit.org/show_bug.cgi?id=267791</a>
<a href="https://rdar.apple.com/121286687">rdar://121286687</a>

Reviewed by Aditya Keerthi.

This patch allows Xcode to index the EventNames.json file, which results
in - among other things - the event names showing up in Xcode search
results.

* Source/WebCore/WebCore.xcodeproj/project.pbxproj:

Xcode was also nice enough to sort the file hashes, so the
JSMediaStreamTrackProcessor entries moved around a bit.

Canonical link: <a href="https://commits.webkit.org/273256@main">https://commits.webkit.org/273256@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/329f20a292eead33840c636ff96da48344ba7d82

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/34854 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/13691 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/36880 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/37552 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/31449 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/36001 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/16095 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/10773 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/30397 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/35384 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/11605 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/31036 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/10147 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/10211 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/31121 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/38809 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/31667 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/31450 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/36237 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/10310 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/8221 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/34210 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/12132 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/10854 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/4485 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/11196 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

pxlcoder commented on Source/WebCore/WebCore.xcodeproj/project.pbxproj:
```diff
@@ -21234,6 +21235,7 @@
 		0867D691FE84028FC02AAC07 /* WebCore */ = {
 			isa = PBXGroup;
 			children = (
+				33E03AFE2B59F557002992E3 /* EventNames.json */,
```
This should be in `dom`, not the top-level `WebCore`.

aprotyas replied:
Hmm, I let Xcode do its thing... I'll try again!

aprotyas replied:
Fixed in 329f20a

aprotyas committed:
EventNames.json should be part of the WebCore Xcode project
https://bugs.webkit.org/show_bug.cgi?id=267791
rdar://121286687

Reviewed by Aditya Keerthi.

This patch allows Xcode to index the EventNames.json file, which results
in - among other things - the event names showing up in Xcode search
results.

* Source/WebCore/WebCore.xcodeproj/project.pbxproj:

Xcode was also nice enough to sort the file hashes, so the
JSMediaStreamTrackProcessor entries moved around a bit.

Canonical link: https://commits.webkit.org/273256@main
